thesslstore_module.php

```diff
@@ -3719,8 +3800,86 @@ private function parseResponse($response, $ignore_error = false) {
         return $response;
     }
 
+    /**
+     * Returns all fields to display to an admin attempting to edit a service with the module
+     *
+     * @param stdClass $package A stdClass object representing the selected package
+     * @param $vars stdClass A stdClass object representing a set of post fields
+     * @return ModuleFields A ModuleFields object, containg the fields to render as
+     *  well as any additional HTML markup to include
+     */
+    public function getAdminEditFields($package, $vars = null)
+    {
+        Loader::loadHelpers($this, ['Html']);
 
+        $fields = new ModuleFields();
 
+        // Create domain label
+        $thesslstore_fqdn = $fields->label(Language::_('ThesslstoreModule.tab_client_cert_details.domains', true), 'thesslstore_fqdn');
+        // Create domain field and attach to domain label
+        $thesslstore_fqdn->attach(
+            $fields->fieldText('thesslstore_fqdn', $this->Html->ifSet($vars->thesslstore_fqdn), ['id'=>'thesslstore_fqdn'])
+        );
+        // Set the label as a field
+        $fields->setField($thesslstore_fqdn);
+
+        // Create order id label
+        $thesslstore_order_id = $fields->label(Language::_('ThesslstoreModule.tab_client_cert_details.store_order_id', true), 'thesslstore_order_id');
+        // Create order id field and attach to order id label
+        $thesslstore_order_id->attach(
+            $fields->fieldText('thesslstore_order_id', $this->Html->ifSet($vars->thesslstore_order_id), ['id'=>'thesslstore_order_id'])
+        );
+        // Set the label as a field
+        $fields->setField($thesslstore_order_id);
+
+        // Create token label
+        $thesslstore_token = $fields->label(Language::_('ThesslstoreModule.tab_client_cert_details.token', true), 'thesslstore_token');
+        // Create token field and attach to token label
+        $thesslstore_token->attach(
+			$fields->fieldText('thesslstore_token', $this->Html->ifSet($vars->thesslstore_token), ['id'=>'thesslstore_token'])
+        );
+        // Set the label as a field
+        $fields->setField($thesslstore_token);
+
+        return $fields;
+    }
+	
+    /**
+     * Edits the service on the remote server. Sets Input errors on failure,
```

Comment:
This comment is inaccurate.  You only edit fields locally and you never set errors.  Careful with copy paste.